mets.md

```diff
@@ -28,6 +56,40 @@ $> exiftool output.tif |grep 'X Resolution'
 "150"
 ```
 
+## Images and coordinates
+
+Coordinates are always absolute, i.e. relative to extent defined in the `imageWidth`/`imageHeight` attribute of the nearest `<pc:Page>`.
+
+When a processor wants to access the image of a layout element like a TextRegion or TextLine, the algorithm should be:
+
+- If the element in question has an attribute `imageFilename`, resolve this value
+- If the element has a `<pc:Coords>` subelement, resolve by passing the attribute `imageFilename` of the nearest `<pc:Page>` and the `points` attribute of the `<pc:Coords>` element
+
+
+# Font information
+
+The documentation of these features is also stored in the **METS file**. In
+this case, this information is extracted from the page file. 
```

Comment:
"in this case"?

Reference PAGE docs on the subject.

Reply:
As a link?

Reply:
Not a link, a better description can help.